public/wp-content/themes/newsup/single.php

```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```

Comment:
This merge conflict was not resolved properly. You need to pick whether to keep code from "HEAD" or from "Master" and now, both HEAD and Master code is in this file.

Reply:
oh.. ill replace it with the file from master now.